Source/WebCore/bridge/runtime_object.cpp

```diff
@@ -182,23 +179,22 @@ bool RuntimeObject::getOwnPropertySlot(JSObject* object, JSGlobalObject* lexical
             return true;
         }
     }
-        
+
```

Comment:
Please update your IDE to stop doing this. This is frowned upon as it adds unnecessary noise to patches.

Reply:
I actually did it manually... I thought it wouldn't hurt to get rid of these whitespace since this was a small patch. I'll update the PR.